modin/scripts/configure_head_node.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+HOSTNAME=$1
+KEY=$2
+
+ssh -i $2 -o "StrictHostKeyChecking no" $1 << "ENDSSH"
+pip3 install modin jupyter
```

Comment:
We might want to think about this line for the final implementation. Ideally, we would let the user specify the python they want to use (or somehow require it on the $PATH).

For now, I would suggest just doing `python -m pip` so it installs for the python on the $PATH. How does this affect `ray start` and `ray stop`?

Reply:
I'll add this fix. It shouldn't affect `ray start` and `ray stop` unless the user manually installs Ray with another version of python.